packages/core/src/components/forms/numericInput.tsx

```diff
@@ -0,0 +1,407 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+
+import { AbstractComponent, Classes, Keys } from "../../common";
```

Comment:
somewhat awkward to bring a few things in from the index and the rest from individual files. Position, props, and utils are also exported from index.